common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -350,6 +362,17 @@ public class CommonValue {
             + "    }\n"
             + "}";
 
+    public static final String ML_MODEL_CONTROLLER_INDEX_MAPPING = "{\n"
+            + "    \"_meta\": {\"schema_version\": "
+            + ML_MODEL_CONTROLLER_INDEX_SCHEMA_VERSION
```

Comment:
Do we need model id field?

Reply:
It's the doc id. But when `GET` model cotroller it will print up.